README.md

```diff
@@ -255,3 +255,31 @@ class HelloWorldConfigurer(Configurer):
         registry.add_interceptor(HelloWorldInterceptor())
 
 ```
+
+## Undefined JSON fields
+
+By default, if a JSON request contains a field that is not defined in a dataclass, an exception will be thrown.
+To accept missing fields in dataclass, you can use Undefined class to explicitly mark fields as optional.
+
+```python
+import winter
+from dataclasses import dataclass
+from typing import Union
+from winter.core.json import Undefined
+
+
+@dataclass
+class RequestBody:
+    field_a: Union[str, Undefined]
+    field_b: Union[str, Undefined] = Undefined()  # explicit default value is not required
+
+
+class SomeAPI:
+    @winter.route_post('/')
+    @winter.request_body('body')
+    def some_method(self, body: RequestBody):
+        if body.field_a == Undefined():
```

Comment:
maybe we want a predefined instance too?

Reply:
I thought about it and didn't find an elegant way to do it. For example, creating a global instance with a different name (like `undefined`) seems not convenient as it requires importing two names instead of two.

Reply:
What do you think about enum

```python
import enum


class Undefined(enum.Enum):
     INSTANCE = 'Undefined'


if body.field_a is Undefined.INSTANCE:
    ...
```

Reply:
enum seems artificial, more verbose and less intuitive from my point of view